src/extension/tests/helpers/RuntimeComposer.py

```diff
@@ -13,7 +13,6 @@
 class RuntimeComposer(object):
     def __init__(self):
         self.backup_os_getenv = os.getenv
-        os.getenv = self.getenv_telemetry_enabled
```

Comment:
remove self.backup_os_getenv = os.getenv also, if not used and the line which resets os.getenv from backup

Reply:
This line was needed - added back